pediatric_rootlets/pediatric_rootlets.sh

```diff
@@ -128,7 +128,7 @@ sct_qc -i ${file_t2}.nii.gz -s ${file_t2}_label-SC_mask.nii.gz -d ${file_t2}_lab
 label_if_does_not_exist ${file_t2}.nii.gz
 
 # Get centerline of the spinal cord
-sct_get_centerline -i ${file_t2}.nii.gz -c t2
+sct_get_centerline -i ${file_t2}.nii.gz -c t2 -method fitseg
```

Comment:
Maybe you can add a QC for this centerline:

```
sct_qc ... -p sct_get_centerline ...
```

Reply:
Centerline is not needed for `sct_label_utils` - these lines have been removed.